devdox/app/routes/git_tokens.py

```
@@ -205,59 +108,21 @@ async def get_git_label_by_label(
     description="Create a new git hosting service token configuration",
 )
 async def add_git_token(
-    request: Request,
+    service: Annotated[PostGitLabelService, Depends(PostGitLabelService.with_dependency)],
     payload: GitLabelBase = Body(...),
```

Ruff (0.11.9)

Do not perform function call `Body` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)